python/setup.py

```diff
@@ -1,17 +1,18 @@
-# Copyright 2023 XMOS LIMITED.
+# Copyright 2023-2024 XMOS LIMITED.
 # This Software is subject to the terms of the XMOS Public Licence: Version 1.
 from setuptools import setup, find_packages
 
 setup(
     name="sw_pll",
-    version="2.0.0",
+    version="3.2.0",
     packages=["sw_pll"],
     package_dir={
-        "": "python"
+        "": "."
     },
     install_requires=[
         "numpy",
```

Comment:
I think these should be pinned using the `~=` relation, maybe with major and minor, so `numpy~=2.1.*` ? Or maybe just major?